databricks/koalas/plot/matplotlib.py

```diff
@@ -16,7 +16,7 @@
 
 from distutils.version import LooseVersion
 
-import matplotlib
+import matplotlib as mat
```

Comment:
Here is due to the conflict when you import, `from databricks.koalas.plot import matplotlib`
As a module `databricks.koalas.plot.matplotlib` vs `matplotlib` library being imported at `databricks.koalas.plot`. I will fix it back in the next PR that refactors.